### DISCUSSION TITLE
Python套件、模組與類別

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#181] [Q&A] [answered]
套件：英文是package或kit
模組：英文是module
類比：英文是class
我一直搞不清楚，像是numpy、pandas、scikit-learn與Graphviz是哪一個東西？
😵‍💫

**Comment by chia-kai-chang** [comment, marked as answer]
numpy會被認為是library(程式庫)

**Reply by enoch-lightning** [reply]
收到，謝謝老師。由numpy是library(程式庫)來看，pandas、scikit-learn與Graphviz應該也都是library(程式庫)。
